src/app/shared/http.service.ts

```diff
@@ -1,57 +0,0 @@
-import { Injectable, Inject, forwardRef } from '@angular/core';
-import {
-  Headers,
-  Http,
-  Response,
-  RequestOptions,
-  Request,
-  RequestOptionsArgs,
-  XHRBackend
-} from '@angular/http';
-
-import { Observable, of, throwError } from 'rxjs';
-import { catchError, map, tap } from 'rxjs/operators';
-
-import { Broadcaster } from 'ngx-base';
-import { isAuthenticationError } from './check-auth-error';
-
-/**
- * @deprecated
```

Comment:
this is a public library, we can't delete a service without letting it be deprecated for some time

Reply:
@joshuawilson deprecated since 1.4.4, remove in 2.0.0
The advantage of removing it is you can remove the need to do DI on http and HttpClient